modules/swagger-codegen/src/main/resources/swift5/Cartfile.mustache

```diff
@@ -0,0 +1,3 @@
+github "Alamofire/Alamofire" ~> 4.5.0{{#usePromiseKit}}
```

Comment:
@plam4u the version of Alomofire included in podspec doesn't compile with Xcode 11 and Swift 5.1.

I needed to upgrade from `4.5.0` to `4.9.0`.

Can you also update the Cartfile please?